WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -262,6 +262,18 @@ - (BOOL)application:(UIApplication *)app openURL:(NSURL *)url options:(NSDiction
                     }
                 }
             }
+        } else if ([[url host] isEqualToString:@"faq"]) {
+            if ([HelpshiftUtils isHelpshiftEnabled]) {
+                NSString *faqID = [url lastPathComponent];
+
+                UIViewController *viewController = self.window.rootViewController.presentedViewController ?: self.window.rootViewController;
```

Comment:
Thanks, I think the first makes sense in this case. Adding that as an extension.